features/networking/egress-ip.feature

```diff
@@ -869,3 +869,47 @@ Feature: Egress IP related features
     Then the step should succeed
     And the output should contain:
       | Hello OpenShift! |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-46637 
+  @admin
+  @destructive
+  @4.10 @4.9
+  @vsphere-ipi
+  @vsphere-upi
+  Scenario: [Bug 2024880] EgressIP should work when configuring networkpolicy
+    Given I save ipecho url to the clipboard
+    Given I store the schedulable workers in the :workers clipboard
+    And the valid egress IP is added to the "<%= cb.workers[0].name %>" node
```

Comment:
This test egressIP for SDN cluster, not sure if we need cover same egressIP + networkpolicy testing for OVN cluster.

Reply:
yeah, that would be seperate case for OVN (not in this PR) as we know the different implementation between sdn and ovn.

Reply:
/lgtm